.github/workflows/ci-for-diff-ns.yaml

```diff
@@ -58,25 +58,27 @@ jobs:
           kubectl -n test-cluster wait deployment rook-ceph-mon-d --for condition=Available=True --timeout=90s
           kubectl -n test-cluster wait deployment rook-ceph-mon-e --for condition=Available=True --timeout=90s
 
-          # for testing osd purge scale the osd deplyment
-          kubectl --namespace test-cluster scale deploy/rook-ceph-osd-0 --replicas=0
-          # we need to sleep so the osd will be marked down before purging the osd
-          sleep 5
-          kubectl-rook_ceph -o test-operator -n test-cluster rook purge-osd 0 --force
-          kubectl rook_ceph -o test-operator -n test-cluster health
-
           # for testing start-debug and stop-debug deployment
           # mon
           kubectl rook_ceph -o test-operator -n test-cluster debug start rook-ceph-mon-a
-          # sleep till the debug pod get running
-          sleep 5
+          kubectl-rook-ceph.sh wait_for_deployment_to_be_running rook-ceph-mon-a-debug test-cluster
           kubectl rook_ceph -o test-operator -n test-cluster debug stop rook-ceph-mon-a
           # osd
           kubectl rook_ceph -o test-operator -n test-cluster debug start rook-ceph-osd-0
-          # sleep till the debug pod get running
-          sleep 5
+          kubectl-rook-ceph.sh wait_for_deployment_to_be_running rook-ceph-osd-0-debug test-cluster
           kubectl rook_ceph -o test-operator -n test-cluster debug stop rook-ceph-osd-0
 
+           # check if osd 0 is started
+          kubectl-rook-ceph.sh wait_for_deployment_to_be_running rook-ceph-osd-0 test-cluster
+        
+          # for testing osd purge scale the osd deplyment
+          kubectl --namespace test-cluster scale deploy/rook-ceph-osd-0 --replicas=0
+          # we need to sleep so the osd will be marked down before purging the osd
+          sleep 5
+          kubectl-rook_ceph -o test-operator -n test-cluster rook purge-osd 0 --force
```

Comment:
typo?
```suggestion
          kubectl rook_ceph -o test-operator -n test-cluster rook purge-osd 0 --force
```

Reply:
probably it would be fine,
as I discussed this in an issue https://github.com/rook/kubectl-rook-ceph/issues/41#issuecomment-1201185823

Still trying to look more deep into this call.